components/JobTitles/JobTitleInput.tsx

```
@@ -0,0 +1,102 @@
+        onSubmit={handleSubmit(handleAdd)}
+      >
+        <div className="h-[88px] w-full">
+          <Input
+            errorMessage={errors?.jobTitle?.message || ""}
+            isInvalid={!!errors.jobTitle}
+            label="Add job title keyword"
+            labelPlacement="outside"
+            placeholder="Javascript"
+            size="md"
+            type="text"
+            variant={!!errors.jobTitle ? "bordered" : "flat"}
```

Biome

[error] Avoid redundant double-negation.

It is not necessary to use double-negation when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant double-negation


(lint/complexity/noExtraBooleanCast)